ui/src/components/IndividualsTable.vue

```diff
@@ -35,23 +46,102 @@ export default {
   components: {
     IndividualEntry,
     ExpandedIndividual
- },
+  },
   props: {
-    individuals: {
-      type: Array,
+    query: {
```

Comment:
Can we change the name of this? It's not really a query and it's not querying, so it's a bit confusing. Maybe we can rename it to `fetchPage` or `fetch`? I'm not sure what the conventions are for naming props functions in Vue.